2025/build-tooling/demo/6-routes/README.md

```diff
@@ -0,0 +1,48 @@
+# ArcGIS Maps SDK for JavaScript: Fast Development and Build Tooling (Demo App)
+
+[Session Description](../..)
+
+## Technologies used:
+
+- [Calcite Design System](https://developers.arcgis.com/calcite-design-system/)
+- [ArcGIS Maps SDK for JavaScript's ES modules](https://developers.arcgis.com/javascript/latest/)
+- [Vite](https://vitejs.dev/)
+- [TypeScript](https://www.typescriptlang.org/)
+- [ESLint](https://eslint.org/)
+- [React](https://react.dev/)
+- [React Router](https://reactrouter.com/)
+
+## Installation
+
+> [NPM](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm) and [Git](https://git-scm.com/downloads) is required to run the following commands.
+
+1. Clone this sample app
+
+   ```sh
+   git clone https://github.com/maxpatiiuk/esri-dev-summit-presentations esri-dev-summit-presentations
+   cd esri-dev-summit-presentations/2025/build-tooling/demo/final
```

Comment:
Replace `final` with 6-roues?

It looks like I also forgot to do this in a few demos – could you please update those to?